<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherFram</class>
 <widget class="QFrame" name="TeacherFram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="mainLayout">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="profTree">
      <column>
       <property name="text">
        <string>Professeurs</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>331</width>
     <height>499</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="userLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="formW">
      <property name="title">
       <string>Formulaire</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>520</width>
         <height>578</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="formLayout">
        <item>
         <layout class="QFormLayout" name="leftLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <property name="horizontalSpacing">
           <number>4</number>
          </property>
          <property name="verticalSpacing">
           <number>10</number>
          </property>
          <item row="0" column="1">
           <widget class="QCheckBox" name="activeBox">
            <property name="text">
             <string>Activ√©</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="loginTxt"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="passTxt"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Nom</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="nomTxt"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Prenom</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="prenomTxt"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Telephone 1</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="phoneTxt"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Telephone 2</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="phonebisTxt"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Adresse</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="addrTxt"/>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Code Postal</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLineEdit" name="codeTxt"/>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Ville</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLineEdit" name="villeTxt"/>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Pays</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QLineEdit" name="paysTxt"/>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Langue</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QLineEdit" name="languageTxt"/>
          </item>
          <item row="3" column="1">
           <widget class="QCalendarWidget" name="date"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="rightLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <property name="horizontalSpacing">
           <number>3</number>
          </property>
          <property name="verticalSpacing">
           <number>10</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Matiere</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="classTxt"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Classes</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="menuW">
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>571</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="menuLayout">
        <item>
         <widget class="QPushButton" name="AddButton">
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SaveButton">
          <property name="text">
           <string>Sauvegarder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CancelButton">
          <property name="text">
           <string>Annuler</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DelButton">
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
